kedro-org/kedro-viz
# Remove GraphQL subscription

## Description

This PR removes the GraphQL subscription from experiment tracking. This feature was added because it seemed like it would benefit users. We've never heard anyone mention they like this feature, and it causes some annoyances when deploying Viz statically, so we're removing it.

## Development notes

The relevant parts of code have been removed. 

## QA notes

Experiment should still work exactly the same. The only difference would be if you were running a Viz process and then did a `kedro run`, your experiment tracking page wouldn't live reload.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/1554"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
